featured/blogs/mr-blog.ftd

```diff
@@ -5,13 +5,13 @@
 -- ds.featured-category: Midnight Rush
 featured-link: /featured/blogs/
 category: Blogs
-image: $fastn-assets.files.images.featured.doc-sites.midnight-rush.jpg
+image: $fastn-assets.files.images.featured.blog.midnight-rush-blog.jpg
 owners: [$muskan-verma.info, $priyanka-yadav.info]
 license-url: https://github.com/fastn-community/midnight-rush/blob/main/LICENSE
```

Comment:
@priyanka9634 plz change the license template you are using in your account. Don't use FifthTry org licence template.